terraform/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "log_consumer_endpoint" {
+  type = string
+}
+//Typically Log-Archive in the SEA has two buckets to store the logs"
+variable "triggerbucket1" {
```

Comment:
Instead of two separate variables this could be a list or a map. You can then simplify the terraform to use `for_each` to create the resources instead of duplicating the code.